test/performance/specs/front-end-classic-theme.spec.js

```diff
@@ -27,7 +27,7 @@ test.describe( 'Front End Performance', () => {
 	const throwaway = 0;
 	const rounds = samples + throwaway;
 	for ( let i = 1; i <= rounds; i++ ) {
-		test( `Report TTFB, LCP, and LCP-TTFB (${ i } of ${ rounds })`, async ( {
+		test( `Measure TTFB, LCP, and LCP-TTFB (${ i } of ${ rounds })`, async ( {
```

Comment:
Just making this title consistent with the [other one](https://github.com/WordPress/gutenberg/blob/2c2f572e36ef417c1dedd01222c94a12615dcaa2/test/performance/specs/front-end-block-theme.spec.js#L31).